src/main/java/org/example/Order.java

```diff
@@ -0,0 +1,35 @@
+package org.example;
+
+public class Order {
+
+    private String orderName;
+    private int quantity;
+    private int price;
+
+    public Order() {
+    }
+
+    public String getOrderName() {
```

Comment:
Minor:
in general, it's better to place getter and setter for the same field together (like it does IDEA when auto-creating)